packages/valaxy-theme-yun/layouts/post.vue

```diff
@@ -4,8 +4,13 @@ import { useConfig, useFrontmatter, useFullUrl } from 'valaxy'
 
 const config = useConfig()
 const frontmatter = useFrontmatter()
-const url = useFullUrl()
 
+const url = computed(() => {
+  if (config.value.autoPostUrl)
+    return window.location.origin + window.location.pathname
```

Comment:
I don't use window.location to solve the ssr rendering problem. If you use window, it will not be able to statically pre-generate, which is not friendly to seo.